h/static/scripts/directives/thread.coffee

```diff
@@ -132,6 +132,7 @@ ThreadController = [
     # thread filter attached to this thread, it will return true.
     ###
     this.matchesFilter = ->
+      return false if @container?.message?.$orphan # We don't want orphans
```

Comment:
Is this necessary? The conversation roots are hidden by the `shouldShowThread`, which makes sense to me because it's view specific (don't show orphans _in the sidebar_). For the other view controllers, this doesn't apply, so that seems like the only pivot needed. This one can go away, perhaps?